packages/react-ui-ag/src/Gmap/Markers.js

```diff
@@ -51,6 +51,7 @@ export default class Markers extends PureComponent {
           if (onMouseOut) onMouseOut(marker)
         },
         ...this.props.marker(feature),
+        zIndex: isActive ? 2 : 1,
```

Comment:
Can you add a test for this? Basically check to make sure zIndex is 2 when active, 1 when inactive.

Reply:
You're right.  I edited and added tests to be more explicit.